frontends/election-manager/src/screens/print_test_deck_screen.tsx

```diff
@@ -130,59 +176,76 @@ export function PrintTestDeckScreen(): JSX.Element {
       }
     } else {
       setPrecinctIds(generatePrecinctIds(precinctId));
-      setPrecinctIndex(0);
+      setPrintIndex({ precinctIndex: 0, sectionIndex: 0 });
     }
   }
 
-  const onAllRendered = useCallback(
+  const onAllOneSidedMaterialsRendered = useCallback(
+    async (numPages) => {
+      if (!printIndex) {
+        return;
+      }
+
+      await printer.print({ sides: 'one-sided' });
+      await makeCancelable(sleep(numPages * 3000));
+      setPrintIndex({
+        precinctIndex: printIndex.precinctIndex,
+        sectionIndex: 1,
+      });
+    },
+    [printIndex, printer, makeCancelable]
+  );
+
+  const onAllTwoSidedMaterialsRendered = useCallback(
     async (numBallots) => {
-      if (typeof precinctIndex !== 'number') {
+      if (!printIndex) {
         return;
       }
 
       await printer.print({ sides: 'two-sided-long-edge' });
       await logger.log(LogEventId.TestDeckPrinted, currentUserType, {
         disposition: 'success',
-        message: `Test Deck printed for precinct id: ${precinctIds[precinctIndex]}`,
-        precinctId: precinctIds[precinctIndex],
+        message: `L&A package printed for precinct id: ${
```

Comment:
Sticking to our approach of valid intermediate states, we may now want to update our button titles, too, e.g."Print Test Decks" --> "Print L&A Package"

Likewise, we can remove the "Print Test Deck Tally Reports" button from the L&A page